CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 All notable changes to this project will be documented in this file.
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.5.0] - September 30, 2020
```

Comment:
I'm curious if this should be a major version bump if it's a breaking change? Based on my understanding of semver, I would think we'd bump this to `1.0.0`. What are your thoughts?

Reply:
According to [the semver spec](https://semver.org/#spec-item-4), when a release is pre 1.0, then 0.x.y changes can be considered breaking.

Reply:
Ah - ok that's a great point.